SU-HSDO/suhumsci
# high level menu changes

# READY FOR REVIEW

# Summary
- High level menu (mostly mobile) styles based on work actively being done by John H and Kerri
- Intentionally time-boxed for Archaeology (will likely need somewhat of an overhaul once menu patterns are solidified and as we move forward)

# Needed By (Date)
- End of sprint (8/2)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Test menu styles to ensure working OK and that nothing is broken

# Affected Projects or Products
- Archaeology
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-299

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
